gateway/1.4/modules/ROOT/pages/flex-conn-reg-run-up.adoc

```diff
@@ -30,8 +21,8 @@ include::partial$task-reg-run-flex-gateway.adoc[tags=user-docker-heading;reg-com
 
 [[kubernetes]]
 // Register and run with a username and password in a Kubernetes cluster
-//intro
-include::partial$task-reg-run-flex-gateway.adoc[tags=user-k8s-heading;k8s-install-flex-helm-chart-intro-connected;k8s-mode-overview]
+//intro (removes k8s-mode-overview tag)
```

Comment:
removing the k8-mode-overview tag for now bc it points to https://dev-docs-internal.kqa.msap.io/beta-flex-gateway-get-well/gateway/1.4/flex-gateway-k8-overview, which is currently for k8 only; not clear if it is also true for openshift

Reply:
that k8 overvew page relates to openshift clusters, so added it back